### PR TITLE
update metadata for A4E

### PR DESCRIPTION
**Pull request by vinnieangel**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

**Comment by azure-quickstarts** [comment]
@ecfan - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

File: quickstarts/microsoft.logic/logic-app-create/metadata.json [hunk 1]
```diff
@@ -6,18 +6,36 @@
...
-    "isActive": "true"
+    "isA4EActive": "true",
+    "usefulLink": [
+            {
+                "name": "Getting Started",
+                "rank": 1,
+                "url": "https://azure.microsoft.com/en-us/resources/templates/logic-app-create/"
+            },
+            {
+                "name": "Azure Logic App",
```
Comment (ecfan) [inline quickstarts/microsoft.logic/logic-app-create/metadata.json:30]: Incorrect name, should be "Azure Logic Apps".

File: quickstarts/microsoft.logic/logic-app-create/metadata.json [hunk 2]
```diff
@@ -6,18 +6,36 @@
...
   "githubUsername": "jeffhollan",
   "docOwner": "ecfan",
-  "dateUpdated": "2022-04-27",
+  "dateUpdated": "2022-07-28",
   "type": "QuickStart",  
   "tags": {
     "subscriptiontype": "Azure4Student",
+    "id": 21,
+    "rank": 7,
     "resource": "Logic Apps, MySQL",
```
Comment (ecfan) [inline quickstarts/microsoft.logic/logic-app-create/metadata.json:15]: Incorrect name, should be "Azure Logic Apps".

File: quickstarts/microsoft.logic/logic-app-create/metadata.json [hunk 3]
```diff
@@ -6,18 +6,36 @@
...
     "os": "Linux, Logic App",
```
Comment (ecfan) [inline quickstarts/microsoft.logic/logic-app-create/metadata.json:16]: Incorrect name, should be "Azure Logic Apps" or just "Azure" as the field is "os".

**Pull request merged**